bower.json

```diff
@@ -1,22 +1,25 @@
 {
   "name": "device-orientation",
-  "description": "Polymer element to detecting device orientation",
+  "description": "Polymer element to detecting device orientation and motion",
   "authors": [
     "Abdón Rodríguez Davila <a@abdonrd.com>"
   ],
   "keywords": [
     "web-component",
+    "webcomponent",
     "polymer",
     "device",
+    "position",
     "orientation",
+    "motion",
     "sensor"
   ],
-  "license": "https://abdonrd.github.io/LICENSE.txt",
+  "license": "MIT",
   "repository": {
     "type": "git",
     "url": "git://github.com/abdonrd/device-orientation.git"
   },
-  "main": "device-orientation.html",
+  "main": "device-orientation-all.html",
```

Comment:
I home the new `iron-component-page` fix that:

<img width="697" alt="screen shot 2017-07-07 at 17 49 49" src="https://user-images.githubusercontent.com/1007051/27965632-c30e5564-633c-11e7-8f5e-b78a3479e1e7.png">